app/helpers/ternary.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+/**
+ * Returns alt if subject is falsey.
+ */
+export default Ember.Helper.helper(function([ subject, alt ]) {
```

Comment:
The `ternary` helper confused me initially. Ember already supports ternary like syntax in `{{if someTestCondition truthyValue falseyValue}}`, and this is a bit more like `coalesce` (in SQL terms) where it is returning the first non-truthy value.

Maybe this can be tweaked to something like `find-first`, `first-truthy`, `coalesce`, something better?  Also, we could allow arbitrary param args via:

```javascript
export default Ember.Helper.helper(function(params) {
  return params.find((item) => item);
})
```